src/tests/test_meta_blk_mgr.cpp

```diff
@@ -322,12 +327,12 @@ class VMetaBlkMgrTest : public ::testing::Test {
         }
     }
 
-    void do_sb_update(const bool aligned_buf_size) {
+    void do_sb_update(const bool aligned_buf_size, uint64_t size_to_update = 0) {
         ++m_update_cnt;
         uint8_t* buf{nullptr};
         auto overflow = do_overflow();
         if (!aligned_buf_size) { overflow = true; } // for unaligned buf size, let's generate overflow buf size;
-        auto sz_to_wrt = rand_size(overflow, aligned_buf_size);
+        auto sz_to_wrt{size_to_update ? size_to_update : rand_size(overflow, aligned_buf_size)};
```

Comment:
I would prefer consistency with develop branch and do `=` for assignments and use `{}` only for object building.

Reply:
will do